src/test/java/com/st/lms/LmsApplicationTests.java

```diff
@@ -0,0 +1,16 @@
+package com.st.lms;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+public class LmsApplicationTests {
```

Comment:
Future PRs should add significant tests, but unless you're going to add them in this class, this class should be removed.